french/dessine/miel/manger-du-miel.md

```diff
@@ -1 +1,12 @@
-Yami, yami, que c'est bon ce miel! Mais pourquoi est - il si chaud?
\ No newline at end of file
+Yami, yami, que c'est bon ce miel! Mais pourquoi est - il si chaud?
+
+Je me rends rapidement compte que ce miel coule sans interruption, comme une fontaine,
+par une grande fente dans le montant superieur de la porte.
+Si proche de cette cascade de miel, je distingue par transparence l'autre cote de la porte.
+Il semble qu'il y ait une foret a l'exterieur. Je decide de traverser cette paroi de nectar
+d'abeille) et me lance a la decouverte de la foret.
+De nombreux insectes se regalent sur moi. Soudain j'entends un grognement a quelques pas derriere
+les arbres. C'est assez inquietant. Que faire?
+
+Devrais-je prendre mes jambes a mon cou?
+ou bien [m'approcher pour savoir ce qui m'attend dans la futaie?] (https://github.com/jeromecholewa/create-your-own-adventure/blob/jerome/french/dessine/miel/forest/approcher.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.